RealmSwift/LinkingObjects.swift

```diff
@@ -469,7 +509,7 @@ public class LinkingObjectsBase: NSObject, NSFastEnumeration {
     internal let objectClassName: String
     internal let propertyName: String
 
-    private var cachedRLMResults: RLMResults?
+    fileprivate var cachedRLMResults: RLMResults?
```

Comment:
`fileprivate` as a keyword is new in Swift 3. This code is in the Swift 2.2 `#else` block. That's what's suspicious about it 😛